### DISCUSSION TITLE
Windows going transparent when not focused

### DISCUSSION THREAD

**Discussion by mikinis111** [i3/i3#6151] [Q&A] [answered]
https://github.com/i3/i3/assets/49591101/cbc9becd-defc-464a-b6ea-a1887bba8efc
As shown in the video, when my windows are not focused and I do some things in another focused window, they go transparent and I have to move my cursor on them or click them to become visible again. It becomes even worse with floating windows, because when there's some process in the terminal behind it, the window keeps disappearing even when I'm using it. Here's my i3 config, please help me if you can. 
[config.txt](https://github.com/user-attachments/files/16162063/config.txt)

**Comment by orestisfl** [comment, marked as answer]
This is probably your compositor. Are you running picom? Try killing it to confirm it's the one causing it. If yes, play around with its settings e.g. `inactive-opacity`.

**Reply by mikinis111** [reply]
Thanks, Compton wiki faq had the solution
